### DISCUSSION TITLE
[MAYA-111375] MAYA2022 how to fix stereo camera : anaglyph mode

### DISCUSSION THREAD

**Discussion by nznosxz** [Autodesk/maya-usd#1494] [Q&A]
i will use stereoCamera change mode anaglyph 

viewport not change and text

// Error: file: C:/Program Files/Autodesk/Maya2022/scripts/others/lookThroughModelPanel.mel line 29: Invalid path 'StereoPanel'.

![image](https://user-images.githubusercontent.com/85823131/121800319-53813900-cc5b-11eb-950a-ab85447148e3.png)

**Comment by santosd** [comment]
Hello @nznosxz, it looks like you are running into a known issue that we are working on fixing internally. A fix for this should be out in a future release.

The issue is logged internally as MAYA-111375.
